consumer.robot

```diff
@@ -8,18 +8,19 @@ Library     RPA.Tables
 *** Tasks ***
 Consume items
     [Documentation]    Login and then cycle through work items.
-    ${Login OK}=    Run Keyword And Return Status
-    ...    Login
-    IF    ${Login OK}
-        For Each Input Work Item    Handle item
-    ELSE
+    TRY
+        Login
+    EXCEPT    Login failed
```

Comment:
I think this should be bare EXCEPT without any message. We would want to catch all error situations with EXCEPT. I know that in our example we can rely on the error message, but this template should provide best practice and in complex keywords it is not wise to rely on an exact error to happen.

Reply:
As this is a template I would also keep the "catch all" -style.. less questions in support when people just take this and do not know how except really works.

Reply:
I disagree, especially with some of the recent discussions about WDM. My understanding is we are going to be changing how an uncaught error releases a work item. A bare except will catch errors that devs didn't account for and cause the robot to technically not fail, but throw an error. In this situation, the final log will be more difficult to troubleshoot. Also, there are risks with a bare except, for example, it will catch `KeywordNotFoundError` and other syntax errors.